algebraic_connectivity_notebook.ipynb

```diff
@@ -441,10 +469,10 @@
     "    test_acc = 0\n",
     "    for epoch in range(1, num_epochs + 1):\n",
     "        # Perform one pass over the training set and then test on both sets.\n",
-    "        do_train(model, train_loader, optimizer, criterion)\n",
-    "        if epoch % 100 == 1 or epoch == 10000:\n",
-    "            train_loss, train_acc = do_test(model, train_dataset, criterion)\n",
-    "            test_loss, test_acc = do_test(model, test_dataset, criterion)\n",
+    "        train_loss = do_train(model, train_data_obj, optimizer, criterion)\n",
+    "        if epoch % 100 == 0 or epoch == num_epochs:\n",
```

Comment:
Skip evaluation in the first epoch. Add an additional condition if this is desired.